### PR TITLE
Team stats (3 methods) AND Game_Team & Collection classes!

### PR DESCRIPTION
**Pull request by gabichuelas**
@rrabinovitch @sciencefixion @evilaspaas1 

Hey team - 
I completed 3 methods from the Team Statistics list, but also wanted to let y'all know that I created a `GameTeam` class and `GameTeamCollection` class (and tests) to account for the third csv file in our repo `game_teams_fixture.csv`. 

So now, `StatTracker` also has that list available. You can call the full list of `GameTeam` objects from that csv with the method `game_teams` on an instance of `StatTracker`. 

Now you can use that collection too if your methods require it!

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,21 +1,29 @@
 class StatTracker
-  attr_reader :games_path, :teams_path, :game_teams_path
+  attr_reader :games_path, :teams_path, :game_teams_path, :game_collection, :team_collection
+
   def initialize(stat_tracker_params)
     @games_path = stat_tracker_params[:games]
     @teams_path = stat_tracker_params[:teams]
     @game_teams_path = stat_tracker_params[:game_teams]
+    @game_collection = GameCollection.new(@games_path)
```
Comment (gabichuelas) [inline lib/stat_tracker.rb:8]: I moved these up here so that we could call the actual `Collection` separately from `Collection.all` in the methods below.

**Review by araspaas — APPROVED** [review]
additions look great.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (rrabinovitch) [reply lib/stat_tracker.rb:8]: nice - I like how this allows for simplification of the code in the `games` method

File: test/team_test.rb [hunk 2]
```diff
@@ -24,4 +26,12 @@ def test_it_has_attributes
     assert_equal "ATL", @team.abbreviation
     assert_equal "/api/v1/teams/1", @team.link
   end
+
+  def test_it_has_info_hash
+    hash = @team.info
```
Comment (rrabinovitch) [inline test/team_test.rb]: let's change the `hash` variable name to `info` or `info_hash` to make it more descriptive

**Review by rrabinovitch — APPROVED** [review]
rocking it!

File: test/team_test.rb (hunk 2 above)
Comment (gabichuelas) [reply test/team_test.rb]: done!

File: lib/stat_tracker.rb (hunk 1 above)
Comment (sciencefixion) [reply lib/stat_tracker.rb:8]: Agreed. Far easier this way.

**Comment by sciencefixion** [comment]
Thanks for all the work that makes our jobs easier! Well done!

**Pull request merged**